packages/core/core-flows/src/product/workflows/update-collections.ts

```diff
@@ -2,23 +2,33 @@ import { ProductTypes } from "@medusajs/types"
 import {
   WorkflowData,
   WorkflowResponse,
+  createHook,
   createWorkflow,
 } from "@medusajs/workflows-sdk"
 import { updateCollectionsStep } from "../steps"
 
 type UpdateCollectionsStepInput = {
   selector: ProductTypes.FilterableProductCollectionProps
   update: ProductTypes.UpdateProductCollectionDTO
+  additional_data?: Record<string, unknown>
 }
 
 type WorkflowInput = UpdateCollectionsStepInput
 
 export const updateCollectionsWorkflowId = "update-collections"
 export const updateCollectionsWorkflow = createWorkflow(
   updateCollectionsWorkflowId,
-  (
-    input: WorkflowData<WorkflowInput>
-  ): WorkflowResponse<ProductTypes.ProductCollectionDTO[]> => {
-    return new WorkflowResponse(updateCollectionsStep(input))
+  (input: WorkflowData<WorkflowInput>) => {
+    const updatedCollections = updateCollectionsStep(input)
+    const collectionsUpdated = createHook("collectionsUpdated", {
+      selector: input.selector,
+      update: input.update,
```

Comment:
Q: do we have to provide the input `selector|update` here?  what was the thinking? Maybe the update in order to try to find what has changed after update right? but the selector I am not sure it is needed, it is more related to what the user want to receive as the result

Reply:
Same question for the other workflows

Reply:
Yeah. I have no suggestions on this. If `selector` `update` seems redundant then we can remove them. CC @olivermrbl what you think?

Reply:
Fixed